### DISCUSSION TITLE
http://localhost/docs/api-reference/digital-asset-management-dam/custom-metadata-fields/list-all-fields

### DISCUSSION THREAD

**Discussion by giscus** [imagekitio/discussions#27] [Docs]
# http://localhost/docs/api-reference/digital-asset-management-dam/custom-metadata-fields/list-all-fields

This API returns the array of created custom metadata field objects. By default the API returns only non deleted field objects, but you can include deleted fields in the API response.

http://localhost/docs/api-reference/digital-asset-management-dam/custom-metadata-fields/list-all-fields

**Comment by yashaswi-imagekit** [comment]
Heyy
